server/config.ts

```diff
@@ -76,8 +76,8 @@ export default {
         deadline: Number(get('HMPPS_AUTH_TIMEOUT_DEADLINE', 10000)),
       },
       agent: new AgentConfig(Number(get('HMPPS_AUTH_TIMEOUT_RESPONSE', 10000))),
-      apiClientId: get('API_CLIENT_ID', 'clientid', requiredInProduction),
-      apiClientSecret: get('API_CLIENT_SECRET', 'clientsecret', requiredInProduction),
+      apiClientId: get('SIGN_IN_CLIENT_ID', 'clientid', requiredInProduction),
+      apiClientSecret: get('SIGN_IN_CLIENT_SECRET', 'clientsecret', requiredInProduction),
       systemClientId: get('SYSTEM_CLIENT_ID', 'clientid', requiredInProduction),
```

Comment:
We probably want the config variable names to match up too? 
`apiClientId` -> `signInClientId`
`apiClientSecret` -> `signInClientSecret` 
So that usage is consistent?

Reply:
added a fix for this now